lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -361,16 +361,24 @@ private static function get_classes( $attributes ) {
 		$text_decoration       = $attributes['style']['typography']['textDecoration'] ?? null;
 		$text_decoration_class = sprintf( 'has-text-decoration-%s', $text_decoration );
 
+		// is-collapsed class is added to the navigation block when the menu is collapsed.
+		$is_collapsed_class = static::is_always_overlay( $attributes ) ? array( 'is-collapsed' ) : array();
```

Comment:
I'm unsure whether its better to add this in PHP and not use the script, or just let the script handle it all. This is more performant so it's probably better although it does raise the complexity.

Reply:
So this will set the `is-collapsed` class on the front of the site in the initial HTML. That's a good thing. I think the additional complexity is worth while. You could consider adding comments to refer to the relevant part of the view.js script and vica versa to ensure it's obvious.

Reply:
Done in da2006b694